packages/apollo-codegen-swift/src/codeGeneration.ts

```diff
@@ -267,7 +267,7 @@ export class SwiftAPIGenerator extends SwiftGenerator<CompilerContext> {
           );
           fragmentsReferenced.forEach(fragmentName => {
             this.print(
-              swift`.appending(${this.helpers.structNameForFragmentName(
+              swift`.appending("\\n" + ${this.helpers.structNameForFragmentName(
```

Comment:
@designatednerd Sorry for the additional commit after receiving LGTM.

I just found that the current generated code for Swift doesn't have a newline between query and fragment.
Because of missing newlines, the generated operation ID and sent query's sha256 hash always differs.
After this change, operation ID and sha256 will be same so we can treat Persisted Query more easily.

Reply:
The current implementation outputs the following query document as `source` property of `operationIds.json`.
sha256 hash calculated from this source is put into the same `operationIds.json`.

```graphql
query SomeQuery {
    ...SomeFragment
}
fragment SomeFragment {
    id
}
```

However, Apollo iOS sends the following query document (missing newlines).
sha256 hash is calculated from this source at runtime so there's difference between 2 hashes.

```graphql
query SomeQuery {
    ...SomeFragment
}fragment SomeFragment {
    id
}
```

This commit fixes this problem with appending missing newlines.